rero_ils/modules/documents/mappings/v6/documents/document.json

```diff
@@ -189,15 +189,41 @@
         "copyrightDate": {
           "type": "text"
         },
+        "editionStatement": {
+          "type": "object",
+          "properties": {
+            "editionDesignation": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "text",
+                  "analyzer": "global_lowercase_asciifolding"
+                },
+                "language": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "responsibility": {
```

Comment:
spell

Reply:
it is correct: responsibility and not responsability